golang/main.go

```diff
@@ -1,7 +1,29 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+	"math"
+)
 
+func Log(x float64) float64 {
+	base := 5.0
+	return (math.Log(x) / math.Log(base))
+}
+
+func form(x float64) float64 {
+	a := 1.1
+	b := 0.09
+	return ((math.Log10(x*x - 1)) / (Log((a*math.Pow(x, 2) - b))))
```

Comment:
```suggestion
	return math.Log10(math.Pow((x, 2) - 1)) / Log(a*math.Pow(x, 2) - b)
```

![image](https://github.com/ISUCT/Informatics_2023/assets/71152598/1bef1317-a1d2-4446-a98a-04695aa176bc)